File: wwwroot/js/person-view-model.js
var personViewModel;

$(function () {
    personViewModel = {
        id: ko.observable(1),
        firstName: ko.observable(''),
        lastName: ko.observable(''),
        age: ko.observable(0),
        gender: ko.observable(0),
        lastUpdated: ko.observable('')
    };

    ko.applyBindings(personViewModel);

    personViewModel.firstName('Igor');
    personViewModel.lastName('Kamyshev');
    personViewModel.age(23);
    personViewModel.gender(0);
    personViewModel.lastUpdated(currentTime());
});

function currentTime() {
    var currentdate = new Date();
    var datetime =
        currentdate.getDate() + "/"
        + (currentdate.getMonth() + 1) + "/"
        + currentdate.getFullYear() + " "
        + currentdate.getHours() + ":"
        + currentdate.getMinutes() + ":"
        + currentdate.getSeconds();

    return datetime;
}

File: Models/PersonViewModel.cs
using System;

namespace KnockoutJS.Models
{
    public enum Gender
    {
        Male = 0,
        Female = 1
    }

    public class PersonViewModel
    {
        private const string DATE_FORMAT = "mm/dd/yyyy";

        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public int Age { get; set; }

        public Gender Gender { get; set; } = Gender.Male;

        public string LastUpdated { get; set; } = DateTime.Now.ToString(DATE_FORMAT);
    }
}
